src/ensembl/src/content/app/browser/track-panel/track-panel-list/TrackPanelListItem.tsx

```diff
@@ -107,6 +108,14 @@ const TrackPanelListItem = (props: TrackPanelListItemProps) => {
   const drawerViewButtonHandler = () => {
     const viewName = track.track_id;
 
+    if (drawerView !== viewName) {
+      analyticsTracking.trackEvent({
+        category: 'track_drawer',
+        label: viewName,
+        action: 'opened'
+      });
```

Comment:
Are we tracking the UI event of opening a drawer? @ens-st3 ?

Reply:
Yes